src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -1114,6 +1117,69 @@ int esbmc_parseoptionst::doit_incremental()
   return 0;
 }
 
+int esbmc_parseoptionst::doit_incremental_cb()
+{
+  optionst opts;
+  get_command_line_options(opts);
+
+  if(get_goto_program(opts, goto_functions))
+    return 6;
+
+  if(cmdline.isset("show-claims"))
+  {
+    const namespacet ns(context);
+    show_claims(ns, get_ui(), goto_functions);
+    return 0;
+  }
+
+  if(set_claims(goto_functions))
+    return 7;
+
+  // Get the initial context bound
+  unsigned int initial_context_bound =
+    strtoul(cmdline.getval("initial-context-bound"), nullptr, 10);
+
+  // Get max number of context bounds
+  unsigned int max_context_bound =
+    cmdline.isset("unlimited-context-bound")
+      ? UINT_MAX
+      : strtoul(cmdline.getval("max-context-bound"), nullptr, 10);
+
+  // Get the context bound increment
+  unsigned int context_bound_inc =
+    strtoul(cmdline.getval("context-bound-step"), nullptr, 10);
+
+  // Get the unwind bound
+  unsigned unwind = atoi(opts.get_option("unwind").c_str());
+
+  bool unknown = true;
+
+  for(unsigned int context_bound = initial_context_bound;
+      context_bound <= max_context_bound;
+      context_bound += context_bound_inc)
+  {
+    char str[5];
+    snprintf(str, sizeof(str), "%d", context_bound);
+    opts.set_option("context-bound", str);
```

Comment:
why are you using snprintf here? Check how we set the unwind option in methods like `do_base_case`.